onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java

```diff
@@ -40,7 +40,7 @@
 /**
  * The class for the Base Delta Collection Page.
  */
-public class BaseDeltaCollectionPage extends BaseCollectionPage<Item, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
+public class BaseDeltaCollectionPage extends BaseCollectionPage<DeltaResponse, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
```

Comment:
This is a breaking change, we need to adjust this so DeltaCollectionPage continues to return items, not DeltaResponses